torchtext/data/datasets_utils.py

```diff
@@ -318,3 +319,29 @@ def pos(self):
 
     def __str__(self):
         return self.description
+
+
+@functional_datapipe("read_iob")
+class IOBDataPipe(IterDataPipe):
```

Comment:
should we make this class private? I guess it is only meant as internal utility for two specific datasets.

Reply:
That's a good question. I could imagine someone using it externally for a different IOB dataset, but I don't have strong opinions about making it private. Happy to do that.

Reply:
Made it private and standardized the name with the pipe in SQuAD.